foreman/data_refinery_foreman/foreman/management/commands/create_quantpendia.py

```diff
@@ -53,6 +36,42 @@ def create_job_for_organism(organism: Organism):
 
     return job
 
+def build_dataset(organism: Organism):
+    data = {}
+    experiments = Experiment.objects.filter(
+            organisms=organism,
+            technology='RNA-SEQ',
+        )\
+        .distinct()
+
+    for experiment_page in queryset_page_iterator(experiments):
+        for experiment in experiment_page:
+            # only include the samples from the target organism that have quant.sf files
+            experiment_samples = experiment.samples\
+                .filter(is_processed=True, technology='RNA-SEQ')
+            # split the query into two so to avoid timeouts.
+            # assume processed rna-seq samples have a quant.sf file
+            processed_samples_with_quantsf = experiment_samples\
+                .filter(is_processed=True)\
+                .values_list('accession_code', flat=True)
+            # and only check for quant file for unprocessed samples
+            unprocessed_samples_with_quantsf = experiment_samples\
+                .filter(
+                    is_processed=False,
+                    results__computedfile__filename='quant.sf'
+                )\
+                .values_list('accession_code', flat=True)\
+                .distinct()
```

Comment:
Both of these sub-queries are no longer filtering on the organism. Any reason why not?

Reply:
good catch will fix it